#! /bin/sh /usr/share/dpatch/dpatch-run
## 903_rdoc_documents.dpatch by akira yamada <akira@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Some files in the ruby source provide RDoc comments, but are not
## DP: included in a default rdoc run. This patch allows to generate the doc
## DP: for those files as well.
## DP: Debian specific patch

@DPATCH@
diff -urNad ruby1.8-1.8.6~/ext/.document ruby1.8-1.8.6/ext/.document
--- ruby1.8-1.8.6~/ext/.document	2007-02-13 08:01:19.000000000 +0900
+++ ruby1.8-1.8.6/ext/.document	2007-03-13 16:39:20.000000000 +0900
@@ -1,6 +1,8 @@
 # Add files to this as they become documented
 
+bigdecimal/bigdecimal.c
 enumerator/enumerator.c
+gdbm/gdbm.c
 iconv/iconv.c
 nkf/lib/kconv.rb
 nkf/nkf.c
diff -urNad ruby1.8-1.8.6~/lib/cgi/.document ruby1.8-1.8.6/lib/cgi/.document
--- ruby1.8-1.8.6~/lib/cgi/.document	2007-02-13 08:01:19.000000000 +0900
+++ ruby1.8-1.8.6/lib/cgi/.document	2007-03-13 16:40:00.000000000 +0900
@@ -1,2 +1,2 @@
+session
 session.rb
-
